core/vm/jump_table.go

```diff
@@ -47,6 +47,7 @@ type operation struct {
 	valid   bool // indication whether the retrieved operation is valid and known
 	reverts bool // determines whether the operation reverts state (implicitly halts)
 	returns bool // determines whether the operations sets the return data content
+	intern  bool // indicates that the operation is safe to call before paygas
```

Comment:
Is `intern` short for anything?

Reply:
I was going for "internal reads/writes only".

Reply:
I see, that seems reasonable. I think `internal` is short enough it might be worth spelling out for clarity.